index.js

```diff
@@ -521,7 +519,7 @@ const init = params => {
 
   // Disable ssl if wanted for local development
   if (params.sslEnabled === false) {
-    options.sslEnabled = false
+    options.tls = false
```

Comment:
Not completely sure about this, but `sslEnabled` is definitely gone. It seems like `tls` is the new option now.

See: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-rds-data/interfaces/rdsdataclientconfig.html#tls